### PR TITLE
Add basic RPC methods

### PR DESCRIPTION
**Pull request by chrisdunelm**

### CODE REVIEW

File: src/Ast/AST.php [hunk 1]
```diff
@@ -343,6 +399,26 @@ public function toCode(): string
         };
     }
```
Comment (amanda-tarafa) [inline src/Ast/AST.php:401]: nit: docs missing

File: src/Ast/PhpClass.php [hunk 2]
```diff
@@ -72,6 +72,18 @@ public function withMember(PhpClassMember $member): PhpClass
         return $this->clone(fn($clone) => $clone->members = $clone->members->append($member));
     }
 
+    /**
+     * Create a class with addition members.
```
Comment (amanda-tarafa) [inline src/Ast/PhpClass.php:76]: nit: additional?

File: src/Utils/Type.php [hunk 3]
```diff
@@ -38,6 +38,11 @@ public static function string(): Type
         return new Type(null, 'string');
     }
 
+    public static function int(): Type
```
Comment (amanda-tarafa) [inline src/Utils/Type.php:41]: nit: missing docs: The built in int type?

**Review by amanda-tarafa — APPROVED** [review]
LGTM

File: src/Ast/AST.php (hunk 1 above)
Comment (chrisdunelm) [reply src/Ast/AST.php:401]: done

File: src/Ast/PhpClass.php (hunk 2 above)
Comment (chrisdunelm) [reply src/Ast/PhpClass.php]: done

File: src/Utils/Type.php (hunk 3 above)
Comment (chrisdunelm) [reply src/Utils/Type.php:42]: done :)

**Review by chrisdunelm — COMMENTED** [review]
Thanks, all fixed.

**Pull request merged**